### ISSUE TITLE
Incorrect TextMate scope for Switch and Case

### ISSUE THREAD

**Issue by OpalMonkey** [thqby/vscode-autohotkey2-lsp#117]
Looks like the TextMate scope for `Switch` and `Case` statements is `variable.other`. I'm guessing it should probably be `keyword.control` instead.

**Comment by thqby** [comment]
Under what circumstances?

**Comment by thqby** [comment]
Under what circumstances?

**Comment by OpalMonkey** [comment]
Well, that was unexpected. I had multiple uses in multiple files that all presented the same and foolishly assumed that meant it was always the same. When trying to reproduce in a new file just now, I happened to use the auto-generated Switch you include, which worked fine.

Turns out the issue is that they are case-sensitive. I capitalize all my flow control statements (`Switch` vs. `switch`, `Else If` vs. `else if`, etc.). The lower-case versions work fine. Changing the case of any letter breaks it. I have never noticed anything else that has this issue, just Switch and its associated statements.

I'll be sure to do more testing before submitting issues in the future.
